docs/release-notes/v1.50-v1.59.md

```diff
@@ -10,6 +10,224 @@
+
+#### [All Users]
+
+* **Updates in the Study Menu interface: Menu accessible on both sides of the menu**
+
+The Study Menu now offers enhanced access options, making it easier to navigate ODM. Users can access the menu from both the top and right sides of the metadata editor, streamlining data management workflows
+
+![Menu.png](1.59/Menu.png)
+<figcaption>Updated view of the metadata editor. The menu can easily be accessed from the top part of the study, or by clicking on the right side, next to the Export button</figcaption>
+
+* **Updates in the Study Menu interface: File Manager**
+
+The dashboard now includes a dedicated File Manager accessible from the sidebar menu. This feature lets users view and filter files by type, such as Reference Genome, Flow Cytometry group, Library, and Sample Metadata.
+
+* ??? abstract "Important considerations"
+    * Filters will be displayed in ascending order
+    * If no filters are applied, the types selected as default will be Reference Genomes and Dictionaries.
+
+![fileManager.png](1.59/fileManager.png)
+<figcaption>Access the File Manager section by clicking on the top left menu</figcaption>
+
+### Fixed Issues
+
+* **Updates on DELETE and POST Endpoints DELETE endpoints: Eliminating sync POST and DELETE endpoints**
+
+In previous releases, posting and deleting data within ODM was available to all curator users. Given the importance of these functions, access will now be limited to users with specific permissions. Release 1.59 refines the ODM APIs by removing outdated endpoints, making the API functionality more reliable, streamlined, and user-friendly
+
+* ??? abstract "Important considerations"
+    * To import data, users need to use the Job endpoints (e.g. <code>POST /api/v1/jobs/import/variant</code>)
+    * To delete data, users need to access the Manage Data endpoints (<code>DELETE /api/v1/manage-data/data</code>)
+    * Deletion of data is exclusively available to users classified as Administrators
+
+* ??? danger "Endpoints removed"
+    * POST /api/v1/as-curator/expressions/group/{id}
+    * DELETE /api/v1/as-curator/expressions/gct
+    * DELETE /api/v1/as-curator/variants/group/{id}
+    * POST /api/v1/as-curator/variants/vcf
+    * DELETE /api/v1/as-curator/flow-cytometries/group/{id}
+    * POST /api/v1/as-curator/flow-cytometries/facs
+    * DELETE /api/v1/as-curator/preparations/{id}
+    * DELETE /api/v1/as-curator/preparations/group/{id}
+    * POST /api/v1/as-curator/preparations
+    * DELETE /api/v1/as-curator/libraries/{id}
+    * DELETE /api/v1/as-curator/libraries/group/{id}
+    * POST /api/v1/as-curator/libraries
+    * DELETE /api/v1/as-curator/samples/{id}
+    * POST /api/v1/as-curator/samples
+    * DELETE /api/v1/as-curator/studies/{id}
+    * POST /api/v1/as-curator/studies
+
+![OutdatedEndpoints.png](1.59/OutdatedEndpoints.png)
```

Comment:
I'm not sure this screenshot is necessary; a list of the removed endpoints would probably suffice.